features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/Scenario.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Created by Jamie Lynch on 06/03/2018.
+// Copyright (c) 2018 Bugsnag. All rights reserved.
+//
+
+import Foundation
+import Bugsnag
+
+internal class Scenario {
+
+    let config: BugsnagConfiguration
+
+    required init(config: BugsnagConfiguration) {
+        self.config = config
+    }
+
+
+    /**
+     * Sets a NOP implementation for the Session Tracking API, preventing delivery
+     */
+    func disableSessionDelivery() {
```

Comment:
There may be special cases at some time in the future, but for now, there should not be override mechanisms for changing request delivery. If unexpected requests are happening, then either:

* There is a bug which should be fixed.
* The scenario should be made more specific to account for it
* Properties on the `BugsnagConfiguration` should be changed to exclude them (e.g. disabling session tracking)